src/Params/EnvelopeParams.cpp

```diff
@@ -545,6 +577,7 @@ void EnvelopeParams::getfromXML(XMLwrapper& xml)
 
     if(!xml.hasparreal("A_dt")) {
         A_dt = dTREAL(xml.getpar127("A_dt", 0));
+        if (A_dt == 0.0f) A_dt= 0.0001f;
```

Comment:
Why does this constraint only apply to attack

Reply:
guess, I missed the others because they don't have 0 as default value. i will add them.